.github/workflows/rust.yml

```diff
@@ -10,14 +10,8 @@ jobs:
             matrix:
                 include:
                     - rust: stable
-                      env:
-                          RUSTFMTCHK: true
```

Comment:
Very Nit: I don't really have any history in this repo but just wanted to register a vote in favor of keeping `rustfmt`. We have had the discussion before elsewhere so won't repeat but IMHO formatting consistency is more important than disliking some of rustfmt behaviour

Reply:
:upside_down_face: Let's say I did this to see if CI would work without having to already deal with how rustfmt is going to decide how I should have formatted my code.